Snakefile

```diff
@@ -107,6 +107,25 @@ rule all:
                dataset=DATASETS,
                seed=range(0,NUM_SEEDS)
                ),
+        # subset_all sepsis batch effect corrected
+        expand("results/subset_all.sepsis.{supervised}.{dataset}.{seed}.be_corrected.tsv",
```

Comment:
It probably won't be necessary since the `keep_ratios` experiment should be a strictly better gauge of performance. I'll go ahead and add TB to the snakemake file though so it gets run at some point